modules/game_stats.rb

```diff
@@ -14,54 +14,51 @@ def lowest_total_score
     scores.min
   end
 
-  #need home team id, hometeamwin.count / num of games
   def percentage_home_wins
    home_winners = creator.games_hash.values.find_all do |games|
         games.home_team_stat.result == "WIN"
     end
-    home_winners.count.to_f / creator.games_hash.count.to_f.round(2)
+    (home_winners.count.to_f / creator.games_hash.count).round(2)
```

Comment:
Integer division is a killer on all of these methods. Good job switching to float.